sae_analysis/visualizer/data_fns.py

```diff
@@ -462,11 +465,7 @@ def get_all_html(self, debug: bool = False, split_scripts: bool = False) -> str:
         if debug:
             display(HTML(html_string))
 
-        if split_scripts:
-            scripts, html_string = extract_and_remove_scripts(html_string)
-            return scripts, html_string
```

Comment:
This `split_scripts` option causes the return type to be a tuple instead of a string, which breaks typing. It looks like this option is never used anyway throughout the code so I just deleted it here. If this is not OK I can try going back and casting everything to `str` everywhere this function is called, but IMO it's probably cleaner to have functions have only a single return type in general.